index.html

```diff
@@ -0,0 +1,128 @@
+                Maybe
+            </label><br>
+            <label>
+                <input type="radio" name="user-recommend" id="" value="not-sure">
+                Not sure
+            </label><br>
+        </div>
+        <div class="info-div">
+            <label for="favorite-feature">What is your favorite feature of freeCodeCamp?</label><br>
+            <select class="input-class" name="mostlike" id="">
+                <option value="challenges">Challenges</option>
+                <option value="projects">Projects</option>
+                <option value="community">Community</option>
+                <option value="openSource">Open Source</option>
+            </select><br>
+        </div>
+
+        <div class="info-div">
+            <label for="">What would you like to see improved? (Check all that apply)</label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="front-end-projects">
+                Front-end projects
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="back-end-projects">
+                Back-end projects
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="data-visualization">
+                Data Visualization
+            </label> <br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="challenges">
+                Challenges
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="open-source-community">
+                Open Source Community
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="citter-help-rooms">
+                Gitter help rooms
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="videos">
+                Videos
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="city-meetups">
+                City Meetups
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="wiki">
+                Wiki
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="forum">
+                Forum
+            </label><br>
+            <label for="">
+                <input type="checkbox" name="prefer"  value="additional-course">
+                Additional Courses
+            </label><br>
+        
+        </div>
+
+        <div class="info-div">
+            <label for="">Any comments or suggestions?</label><br>
+            <textarea class="input-class" name="" id="" cols="30" rows="10" placeholder="Enter your comment here..."></textarea><br>
+        </div>
```

Comment:
- [ ] kindly consider phrasing your comments  as suggestions instead of orders this will help the person you are reviewing to not feel bad as it will be friendly and more polite
- hint 👇 
> * This is a polite suggestion: *"I think it would be better to use semantic tags"*
> * This reads as an order: *"Use semantic tags"*

Reply:
Fixed.